pkg/util/azureclient/environments.go

```diff
@@ -50,8 +50,8 @@ var (
 		GenevaMonitoringEndpoint: "https://gcs.prod.monitoring.core.windows.net/",
 		AppSuffix:                "aro.azure.com",
 		AppLensEndpoint:          "https://diag-runtimehost-prod.trafficmanager.net/api/invoke",
-		AppLensScope:             "b9a1efcd-32ee-4330-834c-c04eb00f4b33",
-		AppLensTenantID:          "72f988bf-86f1-41af-91ab-2d7cd011db47",
+		AppLensScope:             "0d7b6142-46a3-426a-ad6d-eed97c2a48ee",
+		AppLensTenantID:          "33e01921-4d64-4f8c-a055-5bdaffd5e33d",
```

Comment:
Just checking - we don't need FairFax updated as well bellow?

Reply:
nope, the scope of the changes is for apps authenticated with msft tenant.  They have to authenticate with AME starting from March 13.